### PR TITLE
Release 2.3.4

### PR DESCRIPTION
**Pull request by donnyyung**
Docs changes for 2.3.4

### CODE REVIEW

File: docs/pre-release/releaseNotes.yml [hunk 1]
```diff
@@ -30,6 +30,35 @@ docDescription: >-
 changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELOG.md
 
 items:
+  - version: 2.3.4
+    date: '2021-07-09'
+    notes:
+      - type: bugfix
+        title: Improved IP log statements
+        body: >-
+          Fixed log statements that were printing incorrect IP addresses.
```
Comment (LukeShu) [inline docs/pre-release/releaseNotes.yml:39]: Nit: This isn't quite a complete sentence.

File: docs/pre-release/releaseNotes.yml [hunk 2]
```diff
@@ -30,6 +30,35 @@ docDescription: >-
...
+      - type: bugfix
+        title: Improved messaging when multiple services match a workload
+        body: >-
+          If multiple services matched a workload when performing an intercept, Telepresence would panic.
```
Comment (LukeShu) [inline docs/pre-release/releaseNotes.yml:43]: Nit: I prefer to write "crash" instead of "panic".  IMO it conveys the meaning better to those who aren't familiar with Go.

File: docs/pre-release/releaseNotes.yml [hunk 3]
```diff
@@ -30,6 +30,35 @@ docDescription: >-
...
+          some clusters.
+      - type: bugfix
+        title: Telepresence connect respects pre-existing clusterrole
+        body: >-
+          When Telepresence connects, if the <code>traffic-manager</code>'s desired <code>clusterrole</code> already exists in the
+          cluster, Telepresence will no longer try to update the clusterrole.
+      - type: bugfix
+        title: Helm Chart fixed for clientRbac.namespaced
+        body: >-
+          The Telepresence helm chart no longer fails when installing with `--set clientRbac.namespaced=true`.
```
Comment (LukeShu) [inline docs/pre-release/releaseNotes.yml:60]: 1. "helm" should be capitalized
2. Need to use HTML instead of markdown

File: docs/v2.3/releaseNotes.yml [hunk 4]
```diff
@@ -30,6 +30,35 @@ docDescription: >-
...
+        body: >-
+          Fixed log statements that were printing incorrect IP addresses.
+      - type: bugfix
+        title: Improved messaging when multiple services match a workload
+        body: >-
+          If multiple services matched a workload when performing an intercept, Telepresence would panic.
+          It now gives the correct error message, instructing the user on how to specify which
+          service the intercept should use.
+      - type: bugfix
+        title: Traffic-manger determine's service subnet in own namespace
```
Comment (josecv) [inline docs/v2.3/releaseNotes.yml:47]: This is a little hard to parse, maybe `Traffic Manager creates service in its own namespace to determine service subnet`

File: docs/pre-release/releaseNotes.yml [hunk 5]
```diff
@@ -30,6 +30,35 @@ docDescription: >-
 changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELOG.md
 
 items:
+  - version: 2.3.4
+    date: '2021-07-09'
+    notes:
+      - type: bugfix
+        title: Improved IP log statements
```
Comment (khussey) [inline docs/pre-release/releaseNotes.yml:37]: We should link to the documentation for how to configure the logs for this one and include an image how what the improve IP log statement looks like.

File: docs/pre-release/releaseNotes.yml (hunk 2 above)
Comment (khussey) [inline docs/pre-release/releaseNotes.yml]: We should link this entry to the documentation about intercepts (and workloads) and include an image that shows the correct error message.

File: docs/pre-release/releaseNotes.yml [hunk 6]
```diff
@@ -30,6 +30,35 @@ docDescription: >-
...
+        body: >-
+          Fixed log statements that were printing incorrect IP addresses.
+      - type: bugfix
+        title: Improved messaging when multiple services match a workload
+        body: >-
+          If multiple services matched a workload when performing an intercept, Telepresence would panic.
+          It now gives the correct error message, instructing the user on how to specify which
+          service the intercept should use.
+      - type: bugfix
+        title: Traffic-manger determine's service subnet in own namespace
```
Comment (khussey) [inline docs/pre-release/releaseNotes.yml]: determine's -> determines

File: docs/pre-release/releaseNotes.yml [hunk 7]
```diff
@@ -30,6 +30,35 @@ docDescription: >-
...
+        body: >-
+          Telepresence will now determine the service subnet by creating a dummy-service in its own
```
Comment (khussey) [inline docs/pre-release/releaseNotes.yml:50]: Did we have existing documentation about how Telepresence determines the service subnet? If so, we should update it; if not, we should add it. In both cases we should link to it here.

File: docs/pre-release/releaseNotes.yml [hunk 8]
```diff
@@ -30,6 +30,35 @@ docDescription: >-
...
+          namespace, instead of the default namespace, which was causing RBAC permissions issues in
+          some clusters.
+      - type: bugfix
+        title: Telepresence connect respects pre-existing clusterrole
+        body: >-
```
Comment (khussey) [inline docs/pre-release/releaseNotes.yml:55]: I see that we've made updates to the RBAC page in this PR; should we link to that page here or perhaps another one?

**Review by khussey — CHANGES_REQUESTED** [review]
See comments. In general, I think we've missed the opportunity to improve our documentation (and link to it) based on these changes and a few of them could really benefit from an image that shows an example of how the user would recognize the now corrected behavior.

File: docs/pre-release/releaseNotes.yml (hunk 8 above)
Comment (donnyyung) [reply docs/pre-release/releaseNotes.yml:56]: I can link to the RBAC changes here since they are mentioned, but the RBAC changes in this PR were more associated with the helm chart fixes (but I still think linking to the helm chart there is better)

File: docs/pre-release/releaseNotes.yml (hunk 7 above)
Comment (donnyyung) [reply docs/pre-release/releaseNotes.yml:51]: Will do, the docs were correct, the behavior was not.

**Review by khussey — APPROVED** [review]
LGTM, thanks.
